Sources/Plasma/CoreLib/plFileSystem.cpp

```diff
@@ -84,6 +84,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 ST::string plFileName::GetFileName() const
 {
+    if(fName.ends_with("/"))
+    {
+        return plFileName(fName.before_last('/')).GetFileName();
+    }
```

Comment:
This is kind of silly. plFileSystem does not know how to get the file name if the last path component is a folder. Open to suggestions here - this function is named file so another reasonable change is to make a different function.